tensorrt_llm/executor/worker.py

```diff
@@ -747,16 +747,27 @@ def handle_for_worker(self, responses: List[tllm.Response]) -> None:
             assert response is not None
             queue = self.worker.return_queue(response.client_id)
 
+            # TODO: pass in k
+            prompt_logprobs, logprobs = compute_logprobs(
+                response.result.context_logits,
+                response.result.generation_logits,
+                response.result.output_token_ids[0])  #TODO: need change
+
+            additional_outputs = {
+                "prompt_logprobs": prompt_logprobs,
+                "generation_logprobs": logprobs,
+            }
+
             # For AsyncQueue.sync_q, we will batch the events to avoid too many
             # event notifications, thus put without wait here.
             if isinstance(queue, _SyncQueue):
                 global_tracer().log_instant("worker-rsp.put")
-                queue.put_nowait(response)
+                queue.put_nowait((response, additional_outputs))
```

Comment:
How about making a Namedtuple for this so that we can add the typing for related variables? This could be better for reading and IDE jumping.